crates/holochain_persistence_lmdb/src/txn/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod lmdb;
+pub use self::lmdb::*;
```

Comment:
what is the point this sub-module? just cut-and-paste everything that's in `src/txn/lmdb.rs` to here. no?

Reply:
This is just following the existing pattern eveywhere in `holochain_persistence`. I think it would be worthwhile to write an issue suggesting a more ergonomic, idiomatic approach across the entire repo?